### ISSUE TITLE
`.numeric-toggle-time` button styles

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#33]
The labels of `.numeric-toggle-time` buttons have 6px right margins (normally to accommodate space between the icon and a checkbox). In this theme, however, this just translates as 6px of extra space on the right side of what should otherwise be a circular button:
<img width="58" height="56" alt="Numeric toggle time button. 6px of extra space on the right inner margin is highlighted" src="https://github.com/user-attachments/assets/c3d90e87-f397-44e2-a85a-406244ecc372" />

**Comment by aviannamiller** [comment]
These also lack keyboard accessibility due to the `display: none` on the actual input component, the checkbox. In other themes, this is mostly a non-issue since the checkboxes are made visible. For the chosen styling here, it would be better markup to make these true button elements (or assign button roles) to bring back that functionality for keyboard users.
